src/test/edu/oregonstate/cs361/battleship/BattleshipModelTest.java

```diff
@@ -40,4 +40,24 @@ public void testCompMiss() {
         BattleshipModel theModel = new BattleshipModel();
         assertFalse(theModel.updateShot("comp", new Coords(5,5)));
     }
+
+    @Test
+    public void compShotTest() {
+        BattleshipModel theModel = new BattleshipModel();
+        assertTrue(theModel.getComputerHits().size() == 0 && theModel.getComputerMisses().size() == 0);
+
+        theModel.updateShot("comp", new Coords(1, 1) );
```

Comment:
Because this shot you're testing isn't random, if you wanted to be more thorough you could actually retrieve the shot from the computerMisses ArrayList and validate that it's located at the specified coordinates (1,1).